db/blob/blob_file_builder.cc

```diff
@@ -104,6 +104,10 @@ Status BlobFileBuilder::Add(const Slice& key, const Slice& value,
     return Status::OK();
   }
 
+  Slice etime_slice = Slice(value.data() + value.size() - sizeof(uint64_t), sizeof(uint64_t));
```

Comment:
where is value format? the last 8 bytes is expire time?

Reply:
format is define in pika project, and this pr is only used in pika.